raiden-dapp/src/views/account/UDC.vue

```diff
@@ -103,7 +103,9 @@ const { mapState: mapStateUserDepositContract } = createNamespacedHelpers('userD
 })
 export default class UDC extends Vue {
   amount = utils.formatEther(DEFAULT_MS_REWARD.mul(2));
-  minAmount = +utils.formatEther(DEFAULT_MS_REWARD);
+  minAmount = +utils.formatEther(
+    DEFAULT_MS_REWARD.add(DEFAULT_MS_REWARD.div(UDC_SECURITY_MARGIN_PERCENTAGE)),
```

Comment:
This is wrong. You're adding the original value divided by a variable called `PERCENTAGE`. It only works here because `10%` also happens to be `1/10` of `100%`. Either you use some logic like `orig.add(orig.mul(percentage).div(100))`, or rename it to something like `DIV_FACTOR` or so.

Reply:
Done